packages/office-ui-fabric-react/src/components/Persona/PersonaCoin.tsx

```diff
@@ -34,7 +34,9 @@ const SIZE_TO_PIXELS = {
 };
 
 /**
- * Red is a color that often has a special meaning, so it is considered a reserved color and can only be set with overrides
+ * These colors are considered reserved colors and can only be set with overrides:
+ * Red is a color that often has a special meaning.
```

Comment:
Please add bullets/dots/numbering for red and lightgrey, to make it clearer that this is a list of colors :)